src/index.js

```diff
@@ -49,13 +42,20 @@ class Masq {
     this.loginKey = null
     this.loginUrl = null
 
-    if (signalhubUrl) {
-      config.HUB_URLS = [signalhubUrl]
+    // override config with constructor options
+    this.config = {
+      ..._config,
+      HUB_URLS: _options.hubUrls ? _options.hubUrls : _config.HUB_URLS,
```

Comment:
I think we should use lowercase properties

Reply:
changed to use the same key names in Masq and config